### PR TITLE
Be able to search super-user and administrator positions

### PR DESCRIPTION
**Pull request by anilsonmez-simsoft**
Super user and administrator options are added to the position type filter in advanced search

Closes [AB#502](https://dev.azure.com/ncia-anet/2aa083a5-af3d-44e1-8c7b-6e9e6b124d91/_workitems/edit/502)

#### User changes
- Users can search super user and administrator type positions.

#### Super User changes
- None

#### Admin changes
- None

#### System admin changes
- [x] anet.yml or anet-dictionary.yml needs change
- [ ] db needs migration
- [ ] documentation has changed
- [ ] graphql schema has changed

### Checklist
  - [x] Described the user behavior in PR body
  - [x] Referenced/updated all related issues
  - [x] commits follow a `repo#issue: Title` title format and [these 7 rules](https://chris.beams.io/posts/git-commit/)
  - [x] commits have a [clean history](https://epage.github.io/dev/commits/), otherwise PR may be squash-merged
  - [ ] Added and/or updated unit tests
  - [ ] Added and/or updated e2e tests
  - [ ] Added and/or updated data migrations
  - [ ] Updated documentation
  - [ ] Resolved all build errors and warnings
  - [ ] Opened debt issues for anything not resolved here

### CODE REVIEW

File: src/main/resources/anet-schema.yml [hunk 1]
```diff
@@ -912,6 +912,10 @@ properties:
             additionalProperties: false
             required: [type]
             properties:
+              name:
+                type: string
+                title: The permision name of this field
```
Comment (gjvoosten) [inline src/main/resources/anet-schema.yml:917]: Typo: "permission"

File: src/main/resources/anet-schema.yml [hunk 2]
```diff
@@ -928,6 +932,10 @@ properties:
             additionalProperties: false
             required: [type]
             properties:
+              name:
+                  type: string
+                  title: The permision name of this field
```
Comment (gjvoosten) [inline src/main/resources/anet-schema.yml:937]: Typo: "permission"

File: src/main/resources/anet-schema.yml [hunk 3]
```diff
@@ -928,6 +932,10 @@ properties:
...
+                  description: Used in the UI for the type filter of position,
```
Comment (gjvoosten) [inline src/main/resources/anet-schema.yml:938]: Dangling comma?

File: src/main/resources/anet-schema.yml [hunk 4]
```diff
@@ -912,6 +912,10 @@ properties:
             additionalProperties: false
             required: [type]
             properties:
+              name:
+                type: string
+                title: The permision name of this field
+                description: Used in the UI for the type filter of position,
```
Comment (gjvoosten) [inline src/main/resources/anet-schema.yml:918]: Dangling comma?

**Review by gjvoosten — CHANGES_REQUESTED** [review]

**Review by midmarch — APPROVED** [review]
Awesome, is was looking for that functionality!

Screenshot from 2022-09-06 14-46-05

For the user "position type" maybe confusing as it may not have directly have somehting to do with their (primary) position.

Maybe we can renames this to: "User Type" or "Access Level" or "ANET Privileges".

**Pull request merged**